webui/static/css/style.css

```diff
@@ -1,218 +1,260 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
```

Comment:
I don't think we need this, I wonder if you can delete it, is it still works?

Reply:
After I deleted the imported font and tried running it again, the results still worked, because the font-family took it from the body tag.